2.x

jarednova opened the pull request:
**Ticket**: #1580 

This will track the progress and tests for `2.x` into `master`

jarednova commented on tests/test-timber-menu.php:
```diff
@@ -145,20 +145,56 @@ function testMenuTwigWithClasses() {
+
+		// Menu item with _menu_item_target set to ''
+		$item = $items[2];
+		$this->assertFalse( '_self', $item->target() );
```
@gchtr wanted to check on this test (it's failing right now). Should `$item->target()` register as `_self` or `false` in this scenario?

gchtr replied:
@jarednova See #2019.

gchtr commented on tests/test-timber-post.php:
```diff
@@ -379,6 +379,47 @@ function testPagedContent(){
 			$this->assertEquals($page2, trim(strip_tags( $post->paged_content() )));
 		}
 
+		function testMetaCustomPreFilterDisable(){
```
@jarednova These test show a few errors with 2.x. I’ll make a new pull request to fix them.

jarednova replied:
Blast! I was afraid that might happen

szepeviktor commented:
What a long labor!

jarednova commented:
but a labor of love!

jarednova commented:
@szepeviktor nope, that's an oversight on my part. There was a bad merge conflict I was trying to untangle so I had temporarily disabled. It's now back, thanks for the alert!

gchtr committed:
Merge branch '2.x' into 2.x-merge-master

gchtr committed:
Remove entries already handled by ramsey/composer-install

nlemoine committed:
Merge pull request #2691 from timber/2.x-php-82

gchtr committed:
Merge pull request #2683 from timber/2.x-remove-request-class

gchtr committed:
Merge remote-tracking branch 'origin/2.x' into 2.x-merge-master

# Conflicts:
#	.github/workflows/test.yml

gchtr committed:
Merge pull request #2696 from timber/2.x-merge-master

mcaskill committed:
Add AllowDynamicProperties attribute

Dynamic (non-explicitly declared) properties are deprecated as of PHP 8.2 and are expected to become a fatal error in PHP 9.0.

To reduce noise, both in error logs and test run logs, this commit adds the `#[AllowDynamicProperties]` attribute to Timber's classes.

Ideally, a magic `__set()` method should be added which will require adding a catch-all data property to store the non-declared fields.

mcaskill committed:
Fix ACF integration test

Fixed deprecated variable in string syntax.

gchtr committed:
Merge pull request #2676 from timber/2.x-can-edit

nlemoine committed:
Add CS import rules

- Add FullyQualifiedStrictTypesFixer rule
- Add NoLeadingImportSlashFixer rule
- Add SingleImportPerStatementFixer rule
- Add GlobalNamespaceImportFixer rule
- Fix code

mcaskill committed:
Improve term link compatibility in Term class

Resolves #2699

gchtr committed:
Merge branch '2.x' into 2.x-theme-parent-and-cache

gchtr committed:
Import JsonSerializable

nlemoine committed:
Merge pull request #2701 from mcaskill/2.x-fix-term-link-compatibility

nlemoine committed:
Merge branch '2.x' of github.com:timber/timber into 2.x-cs-imports

nlemoine committed:
Add native function invocation rule

gchtr committed:
Merge pull request #2649 from timber/2.x-gitignore

gchtr committed:
Remove srcset() and img_sizes() from ImageInterface and ExternalImage

gchtr committed:
Remove unneeded tests for ExternalImage

gchtr committed:
Merge pull request #2659 from timber/2.x-phpstan-level-0

gchtr committed:
Merge branch '2.x' into 2.x-phpstan-setup

# Conflicts:
#	src/Post.php

gchtr committed:
Merge pull request #2630 from timber/2.x-phpstan-setup

gchtr committed:
Use custom test theme instead of a WordPress default theme using the logic from #2675

Theme availability is different across WordPress versions. Let’s use a custom test theme instead of relying on a WordPress theme to be available.

gchtr committed:
Merge branch '2.x' into 2.x-external-image

# Conflicts:
#	src/Image.php

gchtr committed:
Fix bug with different theme directories

gchtr committed:
Rename functions

gchtr committed:
Use prepared test themes and fix side effects

gchtr committed:
Merge pull request #2703 from timber/2.x-external-image-fix-tests

gchtr committed:
Merge pull request #2695 from timber/2.x-external-image

gchtr committed:
Merge branch '2.x' into 2.x-theme-parent-and-cache

# Conflicts:
#	bin/install-wp-tests.sh
#	tests/Timber_UnitTestCase.php
#	tests/test-timber-loader.php
#	tests/test-timber-parent-child.php
#	tests/test-timber-site.php
#	tests/test-timber-theme.php

gchtr committed:
Fix bugs with theme tests

gchtr committed:
Merge pull request #2675 from timber/2.x-theme-parent-and-cache

gchtr committed:
Merge branch '2.x' into 2.x-post-import-data

gchtr committed:
Fix get_info() for Image

gchtr committed:
Merge pull request #2674 from timber/2.x-post-import-data

mcaskill committed:
Add filter "timber/menu/items" to Menu

This allows Timber integrations to customize any unexpected menu item object such as WPML's `WPML_LS_Menu_Item`.

Without this filter, any integration that provides a custom nav menu item object will encounter the following PHP error:

> Uncaught TypeError: Timber\Menu::Timber\{closure}(): Return value must be of type Timber\MenuItem, null returned in vendor/timber/timber/src/Menu.php:289

mcaskill committed:
Fix menu item compatibility with WPML

Added a callback on the 'timber/menu/items' hook to catch WPML's custom nav menu item class `WPML_LS_Menu_Item` and convert it into a `WP_Post` to be correctly consumed by `MenuItemFactory`.

Without this filter, we encounter the following PHP error if we add a language switcher to a menu:

> Uncaught TypeError: Timber\Menu::Timber\{closure}(): Return value must be of type Timber\MenuItem, null returned in vendor/timber/timber/src/Menu.php:312
> #0 [internal function]: Timber\Menu->Timber\{closure}(Object(WPML_LS_Menu_Item))
> #1 vendor/timber/timber/src/Menu.php(311): array_map(Object(Closure), Array)

mcaskill committed:
Filter the theme menu locations array to enforce string or integers

See #2576

nlemoine committed:
Merge pull request #2698 from mcaskill/2.x-php82-allow-dynamic-properties

gchtr committed:
Merge pull request #2707 from mcaskill/2.x-fix-menu-theme-locations

gchtr committed:
Merge branch '2.x' into 2.x-cs-imports

# Conflicts:
#	src/Attachment.php
#	src/ExternalImage.php
#	src/FileSize.php
#	src/Image.php
#	src/ImageHelper.php
#	src/Integration/AcfIntegration.php
#	src/Loader.php
#	src/Menu.php
#	src/Post.php
#	src/Theme.php
#	src/Timber.php

gchtr committed:
Merge branch '2.x' into 2.x-use-assert-same

# Conflicts:
#	src/Image.php
#	tests/test-timber-loader.php

gchtr committed:
Re-apply svg dimensions fix

gchtr committed:
Add return type

gchtr committed:
Merge pull request #2670 from timber/2.x-use-assert-same

gchtr committed:
Add a note about removed logic

gchtr committed:
Merge pull request #2664 from timber/2.x-get-term-from-query

gchtr committed:
Merge branch '2.x' into 2.x-cs-imports

# Conflicts:
#	src/ImageDimensions.php
#	src/Post.php
#	src/Term.php

nlemoine committed:
Fix post preview (#2712)

* Fix post preview

The `timber/post/import_data` filter was running on every `Timber::get_post()` which can be used to get related posts, post metas, etc.

Ensure that the preview ID is matching the current post ID.

Levdbas committed:
Make comment neutral.

No need to use his here. I'd like to think of code of something neutral.

gchtr committed:
Merge pull request #2713 from Levdbas/patch-1

mcaskill committed:
Remove initially proposed filter "timber/menuitem/pre_from"

mcaskill committed:
Rename proposed filter "timber/menu/items" to "timber/menu/item_objects"

As suggested by @gchtr.

Co-authored-by: Lukas Gächter <lukas.gaechter@mind.ch>

gchtr committed:
Merge pull request #2705 from mcaskill/2.x-fix-wpml-menu-item

nlemoine committed:
Fix iterable case (#2715)

nlemoine committed:
2.x clean init (#2714)

gchtr committed:
Merge branch '2.x' into 2.x-upgrade-guide

# Conflicts:
#	docs/v2/upgrade-guides/2.0.md

gchtr committed:
Merge pull request #2684 from timber/2.x-upgrade-guide

nlemoine committed:
Improve scheme detection and take advantage of WP native functions

nlemoine committed:
Remove string helpers

This is handled by PHP 8 and can easily be polyfilled

gchtr committed:
Add note to Upgrade Guide

nlemoine committed:
Merge pull request #2720 from timber/2.x-better-scheme-detection

nlemoine committed:
Merge pull request #2719 from timber/2.x-remove-string-helpers

gchtr committed:
Merge branch '2.x' into 2.x-cs-imports

# Conflicts:
#	src/Factory/PostFactory.php
#	src/ImageHelper.php
#	src/Integration/WpmlIntegration.php
#	src/Post.php
#	src/TextHelper.php
#	src/Timber.php

gchtr committed:
Merge branch '2.x' into 2.x-phpstan-level-1

gchtr committed:
Update PHPStan level

gchtr committed:
Merge branch '2.x-phpstan-level-1' into 2.x-phpstan-level-2

# Conflicts:
#	phpstan.neon
#	src/Attachment.php
#	src/Post.php
#	src/Timber.php

gchtr committed:
Fix some more PHPStan issues

stayallive committed:
Fix `PostFactory::is_image` incorrectly using `wp_check_filetype`

The current way to detect an image is to use `wp_attachment_is_image` and use that with `wp_check_filetype` to detect if the file has an expected extension.

However, `wp_check_filetype` is not meant to be used with an URL but with a path instead. So this check fails currently if the URL has query params for example.

The fix is to use `get_attached_file` to get the actual file path instead of the URL matching what `wp_attachment_is_image` is doing.

NReilingh committed:
Update 2.0.md - fix 404 link to setup guide

Reviewer may want to change "Setup Guide" to "Installation Guide" or "Installation Page" or something else if it's more accurate.

nlemoine committed:
Add workflow to check dos links

nlemoine committed:
Merge pull request #2732 from NReilingh/patch-1

Update 2.0.md - fix 404 link to setup guide

nlemoine committed:
Merge pull request #2736 from timber/2.x-ci-docs-link-checker

nlemoine committed:
Docs links checker workflow adjustments

Levdbas committed:
Replace broken links for new internal ones.

Levdbas committed:
removed link to external guide

Levdbas committed:
replaced link in image guide with best alternative

Levdbas committed:
Removed gulp script section

I could not find the guide anymore and Gulp
is not actively maintained anymore.

Levdbas committed:
Updated template hierarchy docs

Added a new link and tried to clear up the documentation.

Levdbas committed:
Image cookbook, alternative link to cdn explaination.

Levdbas committed:
updated link and docs.

We do not have an overview page at the moment.

Levdbas committed:
Update property and method docblocks of theme.

See #1313

gchtr committed:
Merge pull request #2667 from timber/2.x-phpstan-level-1

gchtr committed:
Merge pull request #2743 from Levdbas/docs-link-update

gchtr committed:
Merge branch '2.x' into 2.x-remove-is-image-method

# Conflicts:
#	src/Image.php

gchtr committed:
Fix wording in Upgrade Guide

gchtr committed:
Merge pull request #2669 from timber/2.x-remove-is-image-method

Levdbas committed:
Add @api tag to Helper

Levdbas committed:
Fix internal link

nlemoine committed:
Fix custom field test

ACF functions shouldn't be used outside ACF integration tests.

nlemoine committed:
Merge pull request #2730 from stayallive/patch-1

Fix `PostFactory::is_image` incorrectly using `wp_check_filetype`

nlemoine committed:
Fix truncating tables

nlemoine committed:
Remove audio & video methods from post

Those methods do not really make sense. Since `Post` is kind of a DTO object, it should not extract html content from `post_content`.

nlemoine committed:
Remove unused private property

gchtr committed:
Merge pull request #2746 from timber/2.x-helper-class-api-tag

gchtr committed:
Merge branch '2.x' into 2.x-phpstan-final-constructors

gchtr committed:
Update Upgrade Guide

Levdbas committed:
Fix ob_function() link

Levdbas committed:
Merge pull request #2660 from timber/2.x-phpstan-final-constructors

2.x Make object constructors final

Levdbas committed:
Merge pull request #2747 from timber/2745-link-checker-report

2745 link checker report

Levdbas committed:
Merge pull request #2751 from timber/2.x-remove-unsed-private-property

Remove unused private property

gchtr committed:
Merge pull request #2744 from timber/1313-breaking-change-to-timberthemes-uri-for-child-themes

nlemoine committed:
Refactor file models

- Remove Filesize class
- Refactor ExternalImage, Attachment, Image to add types and getting metadata in a more lazy way
- Add `format_bytes` filter to format raw file size into a human readable file size
- Fix/add tests
- Fix city-museum.jpg by removing exif data which was causing issues in tests

gchtr committed:
Merge branch 'master' into 2.x-merge-master

# Conflicts:
#	.github/CODEOWNERS
#	.github/PULL_REQUEST_TEMPLATE.md
#	README.md
#	lib/Helper.php

gchtr committed:
Remove unused $_options param

gchtr committed:
Merge pull request #2752 from timber/2.x-merge-master

Levdbas committed:
Merge pull request #2754 from timber/2.x-term-build-params

2.x Remove unused param from `Timber\Term::build()`

nlemoine committed:
Use pathinfo for grabbing file extension

nlemoine committed:
Fix update_post_meta args

Levdbas committed:
Merge branch '2.x' into 2.x-phpstan-level-2

nlemoine committed:
Merge pull request #2749 from timber/2.x-fix-test-meta

Levdbas committed:
Merge pull request #2750 from timber/2.x-remove-audio-video

Remove audio & video methods from `Post`

Levdbas committed:
Merge pull request #2668 from timber/2.x-phpstan-level-2

2.x Fix PHPStan issues on level 2

gchtr committed:
Update Upgrade Guide for #2750

gchtr committed:
Fix a bug when the_post hook runs twice on each post in a loop

gchtr committed:
Remove stale todos

Levdbas committed:
Merge pull request #2755 from timber/2.x-upgrade-guide

2.x Update Upgrade Guide for #2750

gchtr committed:
Clean up some todos in tests

Levdbas committed:
Merge pull request #2756 from timber/2.x-the_post-hook

2.x Fix a bug when `the_post` hook runs twice on each post in a loop

gchtr committed:
Remove compatibility with older WordPress versions

gchtr committed:
Remove unneeded test

gchtr committed:
Remove class that does nothing

gchtr committed:
Remove tests for deprecated functionality

gchtr committed:
Fix a test

gchtr committed:
Remove check for non-supported WordPress version

gchtr committed:
Remove deprecated todo

gchtr committed:
Fix a bug when |time_ago didn’t consider timezones correctly

Levdbas committed:
Update date-time.md

Update docs with new example

gchtr committed:
Test whether skipped tests still cause runtime errors

gchtr committed:
#2278 Improve getting started docs

Levdbas committed:
Merge pull request #2761 from timber/2.x-docs-getting-started

2.x Fix some small issues in Getting Started docs

mcaskill committed:
Fix URLHelper methods is_local and is_external

Resolves #2766

Changed:
- Method `is_local()` to find occurrence on URL's hostname instead of the entire URL.
- Method `is_external()` to use `is_local()`.

mcaskill committed:
Replace URLHelper is_local strstr with strict comparison

This will resolve issues where the current host is a subdomain or substring of the URL's host.

gchtr committed:
Merge pull request #2758 from timber/2.x-time-ago

gchtr committed:
Merge branch '2.x' into 2.x-skipped-tests

gchtr committed:
Merge branch '2.x' into 2.x-cs-imports

# Conflicts:
#	src/DateTimeHelper.php
#	src/Factory/PostFactory.php
#	src/Helper.php
#	src/Image.php
#	src/Integration/CoAuthorsPlus/CoAuthorsPlusUser.php
#	src/Menu.php
#	src/PagesMenu.php
#	src/Pagination.php
#	src/Post.php
#	src/PostsIterator.php
#	src/Term.php

gchtr committed:
Fix imports after merge

gchtr committed:
Update ignored revisions

gchtr committed:
#2772 Fix deprecation bug in PHP 8.1 when a float was implicitly converted to int

gchtr committed:
Add MYSQL_USER

gchtr committed:
Downgrade MariaDB

gchtr committed:
Merge pull request #2757 from timber/2.x-todo

gchtr committed:
Try fixing actions for MariaDB 11

gchtr committed:
Merge pull request #2776 from timber/2.x-fix-test-actions

gchtr committed:
Merge branch '2.x' into 2.x-cs-imports

# Conflicts:
#	src/Loader.php
#	src/Timber.php

Levdbas committed:
Merge remote-tracking branch 'origin/2.x' into 2.x-skipped-tests

gchtr committed:
Merge branch '2.x' into 2.x-implicit-conversion-from-float-to-int

gchtr committed:
Round SVG image dimensions

gchtr committed:
Add return types and documentation

gchtr committed:
Shorten returns

gchtr committed:
Fix test assertion

gchtr committed:
Merge branch '2.x' into 2.x-fix-url-helper-is-external

gchtr committed:
Fix some CS issues

gchtr committed:
Add CS fixes to ignored revs

gchtr committed:
Run coding standard checks only with highest or stable dependencies

gchtr committed:
Merge pull request #2779 from timber/2.x-fix-cs

gchtr committed:
Merge branch '2.x' into 2.x-implicit-conversion-from-float-to-int

gchtr committed:
Merge branch '2.x' into 2.x-skipped-tests

gchtr committed:
Merge branch '2.x' into 2.x-cs-imports

# Conflicts:
#	.git-blame-ignore-revs

gchtr committed:
Merge branch '2.x' into 2.x-refactor-file-models

Levdbas committed:
add check to image create functions

gchtr committed:
Merge pull request #2760 from timber/2.x-skipped-tests

gchtr committed:
Merge pull request #2700 from timber/2.x-cs-imports

Levdbas committed:
remove error_log and prefix functions

mcaskill committed:
Improve docblocks of URLHelper methods is_local and is_external

mcaskill committed:
Replace strstr in URLHelper is_external method

Replaced with `str_starts_with()` to check for the double slash and `parse_url()` to check for the scheme.

This will resolve issues where `http` or `//` might be a substring of the URL's path.

mcaskill committed:
Fix shortened return in URLHelper is_local method

Amends f73f5a223156fe965f9629d71a0cede633ae9099

mcaskill committed:
Improve URLHelper is_local method

if `wp_parse_url()` fails to extract or does not find a host name in the given URL parameter, presume that the parameter might be a host name itself, such as `example.org` (see unit test).

mcaskill committed:
Merge branch '2.x' into 2.x-fix-url-helper-is-external

gchtr committed:
Fix test

gchtr committed:
Fix is_external() to work with domains only

gchtr committed:
Update return types and documentation

gchtr committed:
Update method descriptions

gchtr committed:
Merge pull request #2782 from timber/2.x-fix-url-helper-is-external-updates

gchtr committed:
Merge branch '2.x' into 2.x-implicit-conversion-from-float-to-int

gchtr committed:
Fix coding style issue

gchtr committed:
Merge branch '2.x' into 2771-bug-towebp-causes-uncaught-typeerror-imageistruecolor-argument-1-$image-must-be-of-type-gdimage-null-given-error

# Conflicts:
#	src/Image/Operation/ToJpg.php
#	src/Image/Operation/ToWebp.php

gchtr committed:
Merge pull request #2775 from timber/2.x-implicit-conversion-from-float-to-int

nlemoine committed:
Refactor file models

nlemoine committed:
Merge branch '2.x' of github.com:timber/timber into 2.x-refactor-file-models

# Conflicts:
#	src/Attachment.php
#	src/ExternalImage.php
#	src/FileSize.php
#	src/URLHelper.php

nlemoine committed:
Refactor file models: remove file property

nlemoine committed:
Refactor file models: update phpdoc

nlemoine committed:
fix: phpstan issues & use fully qualified imports

nlemoine committed:
Merge pull request #2783 from timber/2.x-fix-phpstan-issues

fix: phpstan issues & use fully qualified imports

Levdbas committed:
Check expected values as suggested by @nlemoine

Levdbas committed:
Update statements to Weak implicit

Co-authored-by: Nicolas Lemoine <nico.lemoine@gmail.com>

gchtr committed:
Merge branch '2.x' into 2.x-refactor-file-models

gchtr committed:
Merge pull request #2753 from timber/2.x-refactor-file-models

gchtr committed:
Merge pull request #2780 from timber/2771-bug-towebp-causes-uncaught-typeerror-imageistruecolor-argument-1-$image-must-be-of-type-gdimage-null-given-error

Levdbas committed:
docs(cheatcheat): Explained theme path, link and uri helpers.

fixes issue #1313

gchtr committed:
Add note about installing the release candidate

Levdbas committed:
docs: Update v2 caching docs

Levdbas committed:
fix: minor typo in Core method docs

Levdbas committed:
fix: overwrite magic __call method in Site.php

niclm committed:
Typo: Changing functions section references $filters instead of $functions

Levdbas committed:
Removing not needed evaluation.

gchtr committed:
Merge pull request #2796 from timber/2.x-docs-rc

gchtr committed:
Merge pull request #2799 from niclm/patch-1

Levdbas committed:
docs: updated plugin support part

jrathert committed:
fix: consider 'fields' when returning terms from query, incl. tests

jrathert committed:
fix: initialize typed properties correctly

gchtr committed:
Merge pull request #2787 from timber/1313-breaking-change-to-timberthemes-uri-for-child-themes-docs

Levdbas committed:
Merge pull request #2805 from timber/upgrade-guide-no-more-plugin-support

docs: updated plugin support part

Levdbas committed:
Added test

gchtr committed:
Merge branch 'master' into 2.x-merge-master

# Conflicts:
#	README.md
#	docs/v1/getting-started/switch-to-composer.md
#	lib/Admin.php
#	readme.txt

gchtr committed:
Normalize composer.json

gchtr committed:
Merge pull request #2821 from timber/2.x-merge-master

nlemoine committed:
fix: Attachment size doc block

nlemoine committed:
Merge pull request #2824 from timber/fix-attachment-size-doc-block

fix: Attachment size doc block

nlemoine committed:
fix: External attachment alt/caption

gchtr committed:
Merge branch 'master' into 2.x

nlemoine committed:
ci: Update workflows

- Split each job into its own file
- Filter paths so Actions runs only when necessary
- Add cs2pr to provide PR author direct feedback
- Add dependabot for GA
- Bump easy-coding-standards to ^12

nlemoine committed:
ci: only run on main branches

nlemoine committed:
fix: setup composite action

gchtr committed:
Merge pull request #2826 from timber/2.x-update-ci

gchtr committed:
Merge branch '2.x' into extimg-patch

gchtr committed:
Revert final object constructors

nlemoine committed:
Merge pull request #2825 from timber/extimg-patch

Extimg-patch

gchtr committed:
Merge pull request #2806 from jrathert/termquery-patch

gchtr committed:
Merge branch '2.x' into v2-site-overwrite-call-method

Levdbas committed:
Merge branch '2.x' into v2-caching-docs-update

Levdbas committed:
Docs: add suggestions by @gchtr

Levdbas committed:
Updated docblocks for $expires

array is also a valid argument type.

Levdbas committed:
last part of performance guide

nlemoine committed:
fix:  exclude tests from PHPStan

gchtr committed:
Merge pull request #2798 from timber/v2-site-overwrite-call-method

gchtr committed:
Merge pull request #2827 from timber/2.x-revert-final-constructors

gchtr committed:
Merge pull request #2828 from timber/2.x-exclude-tests-from-phpstan

gchtr committed:
fix: Update documentation for 2.0 release

gchtr committed:
Fix link

Co-authored-by: Erik van der Bas <erik@basedonline.nl>

gchtr committed:
Merge pull request #2797 from timber/v2-caching-docs-update

gchtr committed:
Prepare docs for 2.x

gchtr committed:
fix: Fix some anchors

gchtr committed:
Revert "fix: Fix some anchors"

This reverts commit b57670de799e1728b031e48b7347f122f6a5f4a5.

gchtr committed:
fix: Add note about PostsIterator and removal of timber/class/posts_iterator filter

Levdbas committed:
docs: update links to docs in php file

Levdbas committed:
docs: doc path changes

Levdbas committed:
docs: update readme

Levdbas committed:
docs: remove video tutorials from readme

Levdbas committed:
docs: update bug_report.yml

gchtr committed:
Fix permalink for installation docs

gchtr committed:
Merge pull request #2835 from timber/2.x-upgrade-guide

gchtr committed:
Merge pull request #2829 from timber/2.x-prepare-release

gchtr committed:
Merge branch '2.x' into update-v1-docs-path

# Conflicts:
#	README.md

gchtr committed:
Fix installation docs paths

gchtr committed:
Merge pull request #2836 from timber/update-v1-docs-path

gchtr committed:
Merge branch 'master' into 2.x

# Conflicts:
#	bin/timber.php
#	lib/Timber.php
#	readme.txt

gchtr committed:
fix: Fix failing tests for WP 6.4

gchtr committed:
fix: Exclude docs folder for PHPStan

gchtr committed:
Merge pull request #2838 from timber/2.x-fix-tests

gchtr commented:
Timber 2.0 is released: https://github.com/timber/timber/releases/tag/2.0.0.

We kept the `2.x` branch and made it the default branch. We’re not going to merge this into `master` (which was renamed to `1.x`). So we can close this pull request.

szepeviktor commented:
One major version bump of Timber equals 5 children from a female 👱🏻‍♀️
